Controller/Payment/Webhook.php

```diff
@@ -1,4 +1,4 @@
-<?php 
```

Comment:
What changes in this file? if not remove from this PR

Reply:
Previously the email already existing issue was fixed in this PR, so this file gets added to this PR.